bin/telcoin-network/src/genesis/create_committee.rs

```diff
@@ -207,15 +200,16 @@ impl CreateCommitteeArgs {
         // write genesis and config to file
         //
         // NOTE: CLI parser only supports JSON format for genesis
-        Config::store_path(data_dir.genesis_file_path(), tn_config.genesis(), ConfigFmt::JSON)?;
-        Config::store_path(config_path, tn_config, ConfigFmt::YAML)?;
+        Config::write_to_path(data_dir.genesis_file_path(), tn_config.genesis(), ConfigFmt::JSON)?;
```

Comment:
There is a reason genesis is JSON not YAML like the other config files?  I don;t think this really matters but a small consistency thing.